src/renderer/lbry.js

```diff
@@ -204,6 +204,20 @@ Lbry.getAppVersionInfo = () =>
     ipcRenderer.send('version-info-requested');
   });
 
+Lbry.getFileTitle = fileInfo => {
+  const { value, metadata, claim_name, name } = fileInfo;
+  if (metadata) {
+    // downloaded claim
+    return metadata.title || claim_name;
```

Comment:
not my problem :stuck_out_tongue:

Reply:
@btzr-io we're trying to kill/reduce `lbry.js` rather than add functionality there. Could this instead be added to either the selector or reducer for claim data?

Reply:
ok I'll remove it from there ^^